data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Migrate MySQL Data of More than 1 TiB to TiDB
+summary: Learn how to migrate data above Terabytes from MySQL to TiDB.
+---
+
+# Migrate MySQL Data of More than 1 TiB to TiDB
+
+When the data volume to be migrated is small, you can easily [use DM to migrate data](/data-migration/migrate-mysql-tidb-less-tb.md), both for full migration and incremental replication. However, because DM imports data at a slow speed (30~50 GiB/h), when the data volume is large, the migration might take a long time.
+
+This document describes how to migrate large volumes of data from MySQL to TiDB using Dumpling and TiDB Lightning. TiDB Lightning's local backend mode imports data at 500 GiB/h. After the full migration is completed, you can replicate the incremental data using DM.
```

Comment:
```suggestion
This document describes how to migrate large volumes of data from MySQL to TiDB using Dumpling and TiDB Lightning. TiDB Lightning's local backend mode can import data upto 500 GiB/h. After the full migration is completed, you can replicate the incremental data using DM.
```
500GiB/h is a ideal speed, the actual speed varies for different table schema and  cluster scale. 😅